diff --git a/node_modules/@strapi/typescript-utils/lib/utils/get-config-path.js b/node_modules/@strapi/typescript-utils/lib/utils/get-config-path.js
index d0d1105..2ed7f49 100644
--- a/node_modules/@strapi/typescript-utils/lib/utils/get-config-path.js
+++ b/node_modules/@strapi/typescript-utils/lib/utils/get-config-path.js
@@ -16,12 +16,12 @@ const DEFAULT_TS_CONFIG_FILENAME = 'tsconfig.json';
  * @return {string | undefined}
  */
 module.exports = (dir, { filename = DEFAULT_TS_CONFIG_FILENAME, ancestorsLookup = false } = {}) => {
-  const dirAbsolutePath = path.resolve(dir);
-  const configFilePath = ts.findConfigFile(dirAbsolutePath, ts.sys.fileExists, filename);
+const dirAbsolutePath = path.resolve(dir);
+ const configFilePath = ts.findConfigFile(dirAbsolutePath, ts.sys.fileExists, filename);
 
-  if (!configFilePath || ancestorsLookup) {
-    return configFilePath;
-  }
+ if (!configFilePath || ancestorsLookup) {
+   return configFilePath;
+ }
 
-  return configFilePath.startsWith(dirAbsolutePath) ? configFilePath : undefined;
+ return configFilePath.startsWith(dirAbsolutePath.replaceAll("\\", "/")) ? configFilePath : undefined;
 };
