diff --git a/node_modules/strapi-plugin-transformer/server/util/isAPIRequest.js b/node_modules/strapi-plugin-transformer/server/util/isAPIRequest.js
index 8e6b676..62f8ede 100644
--- a/node_modules/strapi-plugin-transformer/server/util/isAPIRequest.js
+++ b/node_modules/strapi-plugin-transformer/server/util/isAPIRequest.js
@@ -1,10 +1,9 @@
 'use strict';
 
-const isAPIRequest = (ctx) =>
-	ctx.state &&
-	ctx.state.route &&
-	ctx.state.route.info &&
-	ctx.state.route.info.type === 'content-api';
+const isAPIRequest = (ctx) =>{
+      const check = ctx?.state?.route;
+	  return typeof(check.handler) === "function" || (typeof(check.handler) === "string" && check.handler.indexOf("api::") !== -1)
+};
 
 module.exports = {
 	isAPIRequest,
